docker/production/Dockerfile

```diff
@@ -49,4 +49,4 @@ RUN npx shadow-cljs release app
 # https://www.uvicorn.org/deployment/
 WORKDIR /src/backend
 ENV PYTHONPATH="${PYTHONPATH}:/src"
-CMD gunicorn -k uvicorn.workers.UvicornWorker api:app -b 0.0.0.0:8080
+CMD gunicorn -k uvicorn.workers.UvicornWorker api:app -b 0.0.0.0:80
```

Comment:
I think this will probably break our OpenShift deployment. There is the port 8080 mentioned several times.
https://github.com/fedora-copr/log-detective-website/blob/main/openshift/log-detective.yaml

Reply:
should it be then done the other way around? listen on 8080 and do the mapping docker-compose.yaml?

Reply:
This container is run in OpenShift, and I don't think it will allow us to run on port 80. I will try today, and eventually revert this.